src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int NUMBER_OF_FIGURE = 6;
```

Comment:
The constant 'NUMBER_OF_FIGURE' should be named in a way that reflects its purpose more clearly, such as 'FIGURE_COUNT'.